### DISCUSSION TITLE
Timeline to version 1.0?

### DISCUSSION THREAD

**Discussion by varsis** [moonrepo/moon#460] [General]
I really like the look of moon vs nx but I don't think it's something I can adopt until it's in a stable state. What is the expected timeline to version 1.0?

**Comment by milesj** [comment]
@varsis Great question! I'd say we're almost there. I'm trying to stabilize the API, mainly around configuration, before releasing 1.0. I do have some breaking changes upcoming soon, and those may be the last.

If I were to give a rough timeline, maybe in the next 2-3 months.

**Comment by milesj** [comment]
Posted an issue with a more clear roadmap: https://github.com/moonrepo/moon/issues/491
